docs/content/deployment/overview.mdx

```diff
@@ -13,52 +24,254 @@ width={2986}
+        <a href="/concepts/dagit/dagit">Dagit</a>
+      </td>
+      <td>
+        Dagit serves the user interface and responds to GraphQL queries. It can
+        have one or more replicas.
+      </td>
+      <td>Supported</td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/deployment/dagster-daemon">Dagster daemon</a>
+      </td>
+      <td>
+        The Dagster daemon operates schedules, sensors, and run queuing.
+        Currently, replicas are not supported.
+      </td>
+      <td>Not supported</td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/concepts/code-locations">Code location server</a>
+      </td>
+      <td>
+        Code lcation servers serve metadata about the collection of its Dagster
+        definitions. You can have many code location servers; each server can
+        have one or more replicas.
+      </td>
+      <td>Supported</td>
+    </tr>
+  </tbody>
+</table>
 
-Dagster requires three long-running services. You can just run Dagit and the Dagster Daemon for simple deployments, and Dagit will manage running User Repository Deployment services for you. For larger deployments, the User Repository Deployment services can be run and managed separately.
+**For simple deployments**, you can run Dagit and the Dagster daemon, letting Dagit manage running the code location servers for you.
 
-| Service                                                                       | Description                                                                                                                                    | Required                                                                           |
-| ----------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
-| [Dagit](/concepts/dagit/dagit)                                                | Dagit serves the user interface and responds to GraphQL queries. It can have 1 or more replicas.                                               | Yes                                                                                |
-| [Dagster Daemon](/deployment/dagster-daemon)                                  | The Dagster Daemon operates schedules, sensors, and run queuing. Currently, replicas are not supported.                                        | Required for schedules, sensors, and run queuing                                   |
-| [User Repository Deployments](/concepts/repositories-workspaces/repositories) | User Repository Deployments serve metadata about user jobs. You can have many repository servers, and each server can have 1 or more replicas. | Recommended. Otherwise, Dagit and the Daemon must have direct access to user code. |
+**For larger deployments**, you can run and manage the code location servers separately.
```

Comment:
Let's remove both of these comments.

This is a vestige of the fact that one of the paths for deploying Dagster was to have the Dagster definitions to be co-located in the server that hosts Dagit. We don't really recommend this path other than if you're developing locally.